docs/source/joblib.rst

```diff
@@ -52,6 +52,15 @@ validated parameter search as follows.
    with parallel_backend('dask.distributed', scheduler_host='localhost:8786'):
        search.fit(digits.data, digits.target)
 
+## Nested Parallelism
+
+Note that this code will not work if you distribute it via dask in order to implement nested parallelism. The reason is that each worker will have a fresh copy of sklearn and `from sklearn.externals.joblib import parallel_backend` will not apply to them, resulting in a "KeyError" when trying to distribute, but no problem when running the inner loop locally. The solution is 
+
+.. code-block:: python
+
+    from distributed.joblib import DistributedBackend 
+    register_parallel_backend('distributed', DistributedBackend, make_default=True)
```

Comment:
I believe this is a copy-pastable version of the failure from above?

```python
from dask.distributed import Client
from distributed.joblib import DistributedBackend
from sklearn.model_selection import cross_val_score
from sklearn.svm import SVR
from sklearn.datasets import make_regression

from joblib import parallel_backend, register_parallel_backend

X, y = make_regression()
dask_client = Client()


def function(i):
    register_parallel_backend('dask.distributed', DistributedBackend,
                              make_default=True)
    with parallel_backend('dask.distributed',
                          scheduler_host=dask_client.scheduler.address):
        score = cross_val_score(SVR(kernel='linear'), X, y, cv=2, n_jobs=-1)
    return score


dask_client.gather(dask_client.map(function, range(5)))
```

For me that raise a long traceback ending in

```pytb
/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/pickle.py in save(self, obj, save_persistent_id)
    494             reduce = getattr(obj, "__reduce_ex__", None)
    495             if reduce is not None:
--> 496                 rv = reduce(self.proto)
    497             else:
    498                 reduce = getattr(obj, "__reduce__", None)

TypeError: can't pickle select.kqueue objects
```

Does that look familiar @RokoMijic ?